### PR TITLE
update large footers to have more obvious privacy policy link

### PR DESCRIPTION
**Pull request by tanyaparker**
**Pegasus large before**
image

**Pegasus large after**
image

**Pegasus small before**
image

**Pegasus small after**
image

**Code Studio before**
image

**Code Studio after**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
fyi @poorvasingal

**Comment by poorvasingal** [comment]
I wonder if for Pegasus small, we should put the link above the Code.org, 2015 to be more consistent.  It doesn't look great below the dropdown.  I like the rest of the updates though!

**Comment by ashercodeorg** [comment]
Should the 2015's be 2016's?

File: pegasus/sites.v3/code.org/views/footer.haml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 %footer#pagefooter
   .content
-    %div{:style=>"float:left"}
-      
+    .left{:style=>"float:left; max-width: 80%;"}
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/footer.haml:3]: should we hoist any of this out into a .css file?

File: pegasus/sites.v3/code.org/views/footer.haml [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+      %a.whitefooterlink{:href=>"/privacy", :style=>"color: #ffa400;"} Privacy Policy
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/footer.haml:4]: unfortunately clash of whitefooterlink and non-white colour :)

**Comment by breville** [comment]
lgtm, though apparently circleci caught something.

**Comment by bcjordan** [comment]
Re-running Circle tests on this now that `staging` branch eyes baselines are up to date. If it comes back with just the handful of eyes errors we expect we can accept the baselines, maybe re-run, then merge into staging and monitor for smooth baseline merge.

**Pull request merged**